etna/models/nn/chronos/chronos_bolt.py

```diff
@@ -0,0 +1,139 @@
+from pathlib import Path
+from typing import List
+from typing import Sequence
+
+from etna import SETTINGS
+from etna.datasets import TSDataset
+from etna.models.nn.chronos.base import ChronosBaseModel
+
+if SETTINGS.chronos_required:
+    import torch
+
+_DOWNLOAD_PATH = Path.home() / ".etna" / "chronos-models" / "chronos-bolt"
+
+
+class ChronosBoltModel(ChronosBaseModel):
+    """
+    Class for pretrained chronos-bolt models.
+
+    This model is only for zero-shot forecasting: it doesn't support training on data during ``fit``.
+
+    Official implementation: https://github.com/amazon-science/chronos-forecasting
+
+    Note
+    ----
+    This model requires ``chronos`` extension to be installed.
+    Read more about this at :ref:`installation page <installation>`.
+    """
+
+    def __init__(
+        self,
+        path_or_url: str,
+        encoder_length: int = 2048,
+        device: str = "cpu",
+        dtype: torch.dtype = torch.float32,
+        limit_prediction_length: bool = False,
+        batch_size: int = 128,
+        cache_dir: Path = _DOWNLOAD_PATH,
+    ):
+        """
+        Init Chronos Bolt model.
+
+        Parameters
+        ----------
+        path_or_url:
+            Path to the model. It can be huggingface repository, local path or external url
+            - If huggingface repository, the available models are:
```

Comment:
Now this string renders incorrectly.
I feel that we had the same case with nested list but I can't find this place
- If I have blank line after `Path to the model. It can be huggingface repository, local path or external url`, the string `If huggingface repository, the available models are` is bolt
- If I also have blank line after `If huggingface repository, the available models are:`, the nested list is highlighted.

Reply:
Have you tried wrap lists with blank lines?

Reply:
Yes. List is still highlighted.

Reply:
Fixed, there must be indent with 2 spaces for nested list, not 4.

Reply:
Usually, in Sphynx the indent is 3 spaces.